### PR TITLE
Add Ionic Skeleton Text to Image component(#1wf7518) 

### PR DESCRIPTION
**Pull request by bashu22tiwari**

### CODE REVIEW

File: src/components/Image.vue [hunk 1]
```diff
@@ -1,14 +1,18 @@
 <template>
-  <img :src="imageUrl"/>
+  <img :src="imageUrl" v-if="imageUrl" />
+  <ion-skeleton-text v-else animated></ion-skeleton-text>
```
Comment (ymaheshwari1) [inline src/components/Image.vue:3]: We can use self-closing tag here.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/Image.vue (hunk 1 above)
Comment (bashu22tiwari) [reply src/components/Image.vue]: Improved code readability by using self-closing app.
Thank You sir for reviewing my code.

File: src/components/Image.vue [hunk 2]
```diff
@@ -25,7 +29,7 @@ export default defineComponent({
   data() {
     return {
       resourceUrl: '',
-      imageUrl: require("@/assets/images/defaultImage.png")
```
Comment (adityasharma7) [inline src/components/Image.vue:28]: I think we haven't handled the case for default image.

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/components/Image.vue (hunk 2 above)
Comment (bashu22tiwari) [reply src/components/Image.vue:28]: Handled the case if we do not get the correct image.
Thankyou Sir, for reviewing my code.

File: src/views/Search.vue [hunk 3]
```diff
@@ -5,7 +5,7 @@
         <ion-title>{{ $t("Search") }}</ion-title>
       </ion-toolbar>
     </ion-header>
-    <ion-content :fullscreen="true">
```
[the remark this answers is not in the document]
Comment (bashu22tiwari) [reply src/views/Search.vue:8]: Though Sir, it is an unwanted change, But it helps in enhancing UI experience, as we do not get our right hand side scroll bar under the header section.

Thankyou Sir, for reviewing my code.

File: src/views/Search.vue [hunk 4]
```diff
@@ -32,7 +32,7 @@ import {
   IonInfiniteScroll,
   IonInfiniteScrollContent,
   IonList,
-  IonListHeader
+  IonListHeader,
```
Comment (ymaheshwari1) [inline src/views/Search.vue:35]: Remove the unwanted comma

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Search.vue (hunk 4 above)
Comment (bashu22tiwari) [reply src/views/Search.vue]: Removed Unwanted commas.
Thank you Sir for reviewing my code.

**Review by adityasharma7 — APPROVED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

**Pull request merged**